README.md

```diff
@@ -93,27 +93,39 @@
 
 ------------------------------------------------------------------------
 
-Ivy is both an ML transpiler and a framework, currently supporting JAX,
-TensorFlow, PyTorch, and Numpy.
-
-Ivy unifies all ML frameworks 💥 enabling you not only to **write code
-that can be used with any of these frameworks as the backend**, but also
-to **convert 🔄 any function, model, or library written in any of them to
-your preferred framework!**
-
-You can check out [Ivy as a transpiler](#ivy-as-a-transpiler) and [Ivy
-as a framework](#ivy-as-a-framework) to learn more about this, try out
-Ivy straight away going through the [Setting up Ivy](#setting-up-ivy)
-section, or dive deep into Ivy\'s [Documentation](#documentation) and
-[Examples](#examples)!
-
-If you would like to contribute, you can join our growing
-[Community](#community) 🌍, check out our [Contributing](#contributing)
-guide, and take a look at the [open
-tasks](https://unify.ai/docs/ivy/overview/contributing/open_tasks.html)
-if you\'d like to dive straight in 🧑‍💻
-
-**Let\'s** [unify.ai](https://unify.ai) **together 🦾**
+Ivy is an open-source machine learning framework and transpiler that 
+enables you to: 
+
+- 💥 **Accelerate your development**: this will allow you to make your 
+machine learning model 3x faster than the one you have.
+- 🔄 **Convert your code into any framework**: you can benefit from all 
+the machine learning ecosystems with different and save time from 
+reproducing the same algorithms with ivy. You can connect machine 
+learning frameworks together. 
```

Comment:
this doesn't make much sense to me, maybe something along the lines of `Use and build of top of any model, library, infrastructure or device by converting any code between frameworks.` Or something similar